lib/elastic_apm/config.rb

```diff
@@ -172,8 +175,9 @@ def initialize(options = {})
     attr_reader   :ignore_url_patterns
 
     alias :disable_environment_warning? :disable_environment_warning
-    alias :verify_server_cert? :verify_server_cert
     alias :disable_send? :disable_send
+    alias :instrument_rake? :instrument_rake
+    alias :verify_server_cert? :verify_server_cert
```

Comment:
What does `verify_server_cert` have to do with the rake spy?

Reply:
Just noticed it was missing an alias with `?` at the end